docs/scanners.rst

```diff
@@ -105,6 +105,8 @@ You can use your phone to "scan" documents. The regular camera app will work, bu
 
 On Android, you can use these applications in combination with one of the :ref:`Paperless-ngx compatible apps <usage-mobile_upload>` to "Share" the documents produced by these scanner apps with paperless. On iOS, you can share the scanned documents via iOS-Sharing to other mail, WebDav or FTP apps.
 
+There is a Shortcut on iOS where you can directly share PDF Documents to paperless here: https://www.icloud.com/shortcuts/52ce6292ea4e4a01951e14454b88bb14
```

Comment:
Please update the setup text of the shortcut to something like below (just grammar, capitalization etc nothing major)
`Please enter the Url (with http) for Paperless. No trailing slash` --> `Please enter the URL (with https) for your Paperless install. Do not include trailing slash.`

`Please enter your API-Key (from Admin-Panel)` --> `Please enter Auth token (found in Admin section).`

Reply:
Hi!
I'll update it tomorrow. I think after saving it will generate  a new Link